WORKSPACE

```diff
@@ -107,6 +108,15 @@ git_repository(
     shallow_since = "1582757372 -0800",
 )
 
+http_archive(
```

Comment:
could you move it to package.bzl along with the other devDependencies? (assuming anyone building this workspace from src downstream needs the dep, and for consistency)